src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -544,13 +544,27 @@ def _all_databases(self) -> list[str]:
                 return list(self._iterator(self._external_catalog.listDatabases()))
             return self._include_database
         except Exception as err:  # pylint: disable=broad-exception-caught
+            if "py4j.security.Py4JSecurityException" in str(err):
+                logger.error(
+                    "Failed to list databases due to Py4JSecurityException. "
+                    "Update or reinstall UCX to resolve this issue.",
+                    exc_info=True,
```

Comment:
Fyi, I prefer, as it is more explicit about which error info is logged

```suggestion
                    exc_info=err,
```